src/Snppts/Snippets/MauiMudBlazor.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Snppts.Snippets;
+
+public class MauiMudBlazor : IAmASnippet
+{
+    public string Slug => "MauiBlazorWeb";
+    public string Title => "Maui Mud Blazor";
+
+    public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("kevmoens/MauiBlazorWeb");
+
+    public bool ContainsAndroidSample => true;
+    public bool ContainsiOSSample => true;
+    public bool ContainsUWPSample => false;
+    public Sdk BuiltWith => Sdk.Maui;
+
+    public string Description => "";
+
+    public IAmAnAuthor AuthorInfo => new KevinMoens();
+
+    public IEnumerable<Uri> ImageUris => new List<Uri>
+    {
+        new Uri("https://raw.githubusercontent.com/kevmoens/MauiBlazorWeb/main/example1.png")
```

Comment:
I think this screenshot will mess up the layout, ideally it would be a portrait one at least as a first screenshot because that will be shown on the frontpage, the other ones may vary